### DISCUSSION TITLE
Part 5 CH01-02 ubuntu install-docker-compose.sh

### DISCUSSION THREAD

**Discussion by SanghoonYang** [tedilabs/fastcampus-devops#38] [General]
docker-compose 설치하고 docker-compose 커맨드가 자꾸 오류가 나서 확인해보니 기존 소스에 정의되있는 2.1.1 버전은 url 이 깨진 것 같습니다.

https://github.com/docker/compose/releases/download/2.1.1/docker-compose-Linux-x86_64

링크 확인해보니 버전 url 앞에 v 가 붙게되어있네요.
DOCKER_COMPOSE_VERSION=v2.2.2
로 하니까 잘 되었습니다.

**Comment by posquit0** [comment]
앗.. 제가 오타를 내었네요 ㅠ.ㅠ
제보해주셔서 감사합니다!

최신 코드에 수정하여 반영하였습니다.

https://github.com/tedilabs/fastcampus-devops/commit/f7b1c6f73de2d1b531aa781961b8198ac8e7fcb4
